pyron-engine/src/main/scala/com/cloudentity/pyron/api/ApiHandler.scala

```diff
@@ -382,15 +407,21 @@ object HttpConversions {
     }
   }
 
-  def toOriginalRequest(req: HttpServerRequest, pathParams: PathParams, bodyOpt: Option[Buffer]): OriginalRequest =
+  def toOriginalRequest(req: HttpServerRequest, pathParams: PathParams, bodyOpt: Option[Buffer]): OriginalRequest = {
     OriginalRequest(
-      method      = req.method(),
```

Comment:
Vertical alignment has this problem, that if you add new, longer item, the whole section needs adjusting. I don't have anything against it, but I don't feel like it's worth actively preserving, especially that IntelliJ will very often mess it up when any refactor is done, but it will auto-format without vertical alignment just fine. I'm pretty sure it was IntelliJ reformat. I would rather ignore it for the reasons stated above.